packages/arm-core/src/components/controls/BaseLayerControl.tsx

```diff
@@ -185,71 +184,75 @@ const BaseLayerToggle: React.FC<Props> = ({
   }, [toggleBaseLayerType, selectedLayer])
 
   return (
-    <Wrapper>
-      <ToggleButton
-        variant="blank"
-        title="Wissel tussen luchtfoto's of een topografische kaarten"
-        onClick={handleToggle}
-        layerType={layerTypeForButton}
-      />
-      {baseLayers[toggleBaseLayerType].length > 1 && (
-        <Menu
-          data-test="context-menu"
-          title="Actiemenu"
-          arrowIcon={<Ellipsis />}
-          selectElementForTouchScreen={
-            <>
-              {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-              <label
-                style={{ display: 'none' }}
-                htmlFor="arm-baselayer-toggle-select"
-              >
-                Open menu
-              </label>
-              <ContextMenuSelect
-                name="context-menu"
-                id="arm-baselayer-toggle-select"
-                onChange={(e) => handleChange(e, currentAmsterdamLayers)}
-              >
-                {baseLayers[toggleBaseLayerType].map(({ id, label }) => (
-                  <option key={id} value={id}>
-                    {label}
-                  </option>
-                ))}
-              </ContextMenuSelect>
-            </>
-          }
-          position="bottom"
-        >
-          {baseLayers[toggleBaseLayerType].map(({ id, label, urlTemplate }) => (
-            <StyledContextMenuItem
-              key={id}
-              onClick={(e) => {
-                setSelectedLayer({
-                  ...selectedLayer,
-                  [toggleBaseLayerType]: urlTemplate,
-                })
-                e.currentTarget.blur()
-              }}
-              icon={
-                urlTemplate === selectedLayer[toggleBaseLayerType] ? (
-                  <CheckmarkIcon inline size={12}>
-                    <Checkmark />
-                  </CheckmarkIcon>
-                ) : null
-              }
-            >
-              {label}
-            </StyledContextMenuItem>
-          ))}
-        </Menu>
-      )}
-      <BaseLayer
-        options={options}
-        baseLayer={selectedLayer[toggleBaseLayerType]}
-      />
-    </Wrapper>
+    <Control>
+      <Wrapper>
+        <ToggleButton
+          variant="blank"
+          title="Wissel tussen luchtfoto's of een topografische kaarten"
+          onClick={handleToggle}
+          layerType={layerTypeForButton}
+        />
+        {baseLayers[toggleBaseLayerType].length > 1 && (
+          <Menu
+            data-test="context-menu"
+            title="Actiemenu"
+            arrowIcon={<Ellipsis />}
+            selectElementForTouchScreen={
+              <>
+                {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+                <label
```

Comment:
Yeah I am not to happy with this component either, I have tried to keep this as much the same as it was before because I don't want to modify this component too much.

Reply:
I thought about this but I don't think it's a very solid argument. This label component seems obsolete. What is the use?